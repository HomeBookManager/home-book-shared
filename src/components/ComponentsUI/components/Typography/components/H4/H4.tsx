import { FC } from 'react';

// components
import Typography from '../../Typography';

// types
import { TypographyFontType } from '../../enums';
import { TOmittedTypographyProps } from '../../types';

const H4: FC<TOmittedTypographyProps> = ({ children, ...restProps }) => (
  <Typography fontType={TypographyFontType.h4} {...restProps}>
    {children}
  </Typography>
);

export default H4;
