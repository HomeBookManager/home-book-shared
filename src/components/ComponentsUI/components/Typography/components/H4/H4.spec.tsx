import { render } from '@testing-library/react';

// components
import H4 from './H4';

describe('H4 snapshots', () => {
  it('should render H4', () => {
    //before
    const { asFragment } = render(<H4>H4</H4>);

    // result
    expect(asFragment()).toMatchSnapshot();
  });
});
