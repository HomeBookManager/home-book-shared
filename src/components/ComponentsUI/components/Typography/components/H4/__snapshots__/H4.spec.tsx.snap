// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H4 snapshots should render H4 1`] = `
<DocumentFragment>
  <h4
    class="Typography Typography--regular Typography--normal Typography--h4 Typography--without-margin"
    data-test-text=""
    style="color: rgb(30, 40, 57);"
  >
    H4
  </h4>
</DocumentFragment>
`;
